app/components/Poll/index.js

```diff
@@ -163,11 +175,47 @@ class Poll extends React.Component<Props, State> {
           </Flex>
         )}
         {!hasAnswered && (
-          <Flex column className={styles.optionWrapper}>
+          <Flex
+            column
+            className={styles.optionWrapper}
+            onMouseEnter={this.toggleHover}
+            onMouseLeave={this.toggleHover}
+            style={{}}
+          >
+            {!expanded && (
+              <Flex
+                style={{ display: isHovering ? 'flex' : 'none' }}
+                className={styles.blurContainer}
+                onClick={this.toggleTruncate}
+              >
+                <p
+                  style={{
+                    marginTop: '25px'
+                  }}
+                  className={styles.blurOverlay}
```

Comment:
Better if you just use `className` here. If you want to apply custom styling for this element, you can use [classnames](https://github.com/JedWatson/classnames) to apply multiple classnames. See f.ex [here](https://github.com/webkom/lego-webapp/blob/60aae52f29b89f1f7bd23150d952df69b38ac090/app/components/Form/RadioButton.js#L43) for an example.
(In general, try to avoid inline styling)

[PS :mega:](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords)